webui/react/src/shared/components/Avatar/Avatar.tsx

```diff
@@ -50,7 +50,7 @@ const Avatar: React.FC<Props> = (
   if (large) classes.push(css.large);
   if (className) classes.push(className);
   const avatar = (
-    <div className={classes.join(' ')} id="avatar" style={style}>
+    <div className={classes.join(' ')} data-testid="avatar-element" id="avatar" style={style}>
```

Comment:
nit: is the change just to facilitate the tests? I'd guess we had added `id` for the same purpose. maybe we don't need both?

Reply:
oh yeah. I couldn't see the `id` 😄

Reply:
👍🏼  I don't know which one is better to keep/use.

Reply:
I think `id` is better in this case? https://stackoverflow.com/questions/55151142/react-testing-library-why-use-test-id